Casks/e/es-de.rb

```diff
@@ -0,0 +1,25 @@
+cask "es-de" do
+  arch arm: "184126565", intel: "184126636"
+
+  version "3.2.0"
+  sha256 :no_check
+
+  url "https://gitlab.com/es-de/emulationstation-de/-/package_files/#{arch}/download",
+      verified: "gitlab.com/es-de/emulationstation-de/"
+  name "ES-DE"
+  name "EmulationStation Desktop Edition"
+  desc "Frontend for browsing and launching games from your multi-platform collection"
+  homepage "https://www.es-de.org/"
+
+  no_autobump! because: :incompatible_version_format
```

Comment:
@samford May have a better suggestion here, but here's a working livecheck.

```suggestion
  livecheck do
    url "https://gitlab.com/api/v4/projects/18817634/repository/files/latest_release.json/raw?ref=master"
    regex(%r{/package_files/(\d+)/download}i)
    strategy :json do |json|
      version = json.dig("stable", "version")
      next if version.blank?

      download_url = json.dig("stable", "packages")
                         &.find { |pkg| pkg["name"] == arch }
                         &.dig("url")
      next if download_url.blank?

      download_id = download_url[regex, 1]
      next if download_id.blank?

      "#{version},#{download_id}"
    end
  end
```